client/src/Profile.css

```diff
@@ -0,0 +1,291 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
```

Comment:
i think we already have this import in `App.css` so this won't be necessary, @leebianca can confirm though